.guides/setup.md

```diff
@@ -0,0 +1,74 @@
+# Firebase Functions Setup Guide
+
+This guide provides a step-by-step process for setting up a new Firebase Functions project, tailored for a coding agent.
+
+### **Step 1: Initialize Firebase Project**
+
+If you haven't already, initialize your Firebase project. The agent should have the `firebase-tools` CLI installed.
+
+```bash
+# One-time setup for a new project
+firebase login
+firebase init functions
+```
+During initialization, select `TypeScript`.
+
+### **Step 2: Install Dependencies**
+
+Navigate to the `functions` directory and install the necessary NPM packages.
+
+```bash
+cd functions
+npm install
+```
```

Comment:
This is an interesting problem I probably need to think more about...the tooling is built to detect guidance of dependencies in the current workspace, but this library is installed in the `functions` subdirectory.

I think you might for now just assume that they've already run `firebase init functions` and have now started a coding agent inside the `functions` dir.

Reply:
🐔 🥚

Reply:
I've removed these steps